lib/platform/bitbucket-server/bb-got-wrapper.ts

```diff
@@ -28,7 +28,10 @@ export const api: GotApi = {} as any;
 
 for (const x of helpers) {
   (api as any)[x] = (url: string, opts: any): Promise<GotResponse> =>
-    get(url, { ...opts, method: x.toUpperCase() });
+    get(url, {
+      ...opts,
+      method: x.toUpperCase(),
+    });
```

Comment:
@daekene Can you please remove this unrelated change.

@rarkins was this prettier?